### PR TITLE
intacct fivetran dag

### PR DESCRIPTION
**Pull request by spatilmoz**

### CODE REVIEW

File: dags/intacct_historical.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  "uk": "toy_tribute"
+}
+
+with DAG('fivetran_intacct_historical',
+         default_args=default_args,
+         doc_md=docs,
+         schedule_interval="0 5 * * *") as dag:
+
+    fivetran_sensors_complete = DummyOperator(
+        task_id='end',
```
Comment (ANich) [inline dags/intacct_historical.py:56]:
```suggestion
        task_id='intacct-fivetran-sensors-complete',
```

File: dags/intacct_historical.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                fivetran_conn_id='fivetran',
+                connector_id='{}'.format(y)
+        )
+
+        fivetran_sync_wait = FivetranSensor(
+                task_id='intacct-sensor-{}'.format(x),
+                fivetran_conn_id='fivetran',
+                connector_id='{}'.format(y),
+                poke_interval=5
+        )
```
Comment (ANich) [inline dags/intacct_historical.py:72]: Just for comprehension:

```suggestion
    for location, connector_id in list_of_connectors.items():

        fivetran_sync_start = FivetranOperator(
                task_id='intacct-task-{}'.format(location),
                fivetran_conn_id='fivetran',
                connector_id='{}'.format(connector_id)
        )

        fivetran_sync_wait = FivetranSensor(
                task_id='intacct-sensor-{}'.format(location),
                fivetran_conn_id='fivetran',
                connector_id='{}'.format(connector_id),
                poke_interval=5
        )
```

**Review by ANich — APPROVED** [review]
LGTM, left some suggestions

**Comment by spatilmoz** [comment]
> LGTM, left some suggestions

comments added and merging further. Thank you so much !!!

**Pull request merged**